ArcGISRuntimeSDKQt_CppSamples/Routing/RouteAroundBarriers/RouteAroundBarriers.json

```diff
@@ -0,0 +1,14 @@
+{
+    "source" :  "RouteAroundBarriers.qml",
+    "otherSourceFiles" :  [
+        "RouteAroundBarriers.h",
+        "RouteAroundBarriers.cpp"
+    ],
+    "name" : "Route around barriers",
+    "dataItems" : [        
```

Comment:
same comment as in the readme - I think it would be preferable to embed as a resource in QRC instead to reduce dependencies to run the sample